### PR TITLE
test: add test job for Go 1.14

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,6 +31,8 @@ matrix:
     env: GOFLAGS=-mod=vendor GO111MODULE=on
   - go: "1.13.x"
     env: GOFLAGS=-mod=vendor
+  - go: "1.14.x"
+    env: GOFLAGS=-mod=vendor
   - go: "tip"
     env: GOFLAGS=-mod=vendor
   - go: "1.13.x"
```
Comment (QuentinBrosse) [inline .travis.yml]:
```suggestion
  - go: "1.14.x"
```

and same for `GOARCH=amd64`

Comment (QuentinBrosse) [reply .travis.yml]: > same for GOARCH=amd64

**Comment by Hades32** [comment]
Why do you even have tests for old Go versions? Why not _only_ for the version that you actually use to build the binaries that you release?

**Comment by jerome-quere** [comment]
We try to follow Go version support policy https://golang.org/doc/devel/release.html#policy

**Comment by marco-m** [comment]
This should be updated for Go 1.15 :-)

**Comment by remyleone** [comment]
https://github.com/scaleway/terraform-provider-scaleway/pull/571/

**Pull request closed without merging**